eessi_bot_job_manager.py

```diff
@@ -125,9 +125,9 @@ def get_current_jobs(self):
         }
 
         # get job info, logging any Slurm issues
-        # Note, the first two lines of the output are skipped ("range(2,...)")
-        #     because they contain header information.
-        for i in range(2, len(lines)):
+        # Note, all output lines of squeue are processed because we run it with
+        # --noheader.
+        for i in range(0, len(lines)):
```

Comment:
Wouldn't `for line in lines` be cleaner then?

Reply:
Yep.

Reply:
Done in https://github.com/EESSI/eessi-bot-software-layer/pull/220/commits/448dcfadc14cbaf14b923bff22b5a238dcfce019